### ISSUE TITLE
Renaming with genre before series splits up series

### ISSUE THREAD

**Issue by mkurios** [tinyMediaManager/tinyMediaManager#438]
__What TMM version are you using?__
2.9.14

__What is the actual behaviour?__
when i put the renamer for movie folders on genre\movieset\movie like format...  as such:
{$G\}{_Collections\$N\}$T { - $U }($Y)
I get moviesets that are split (some movies will be in adventure, some in action, some in etc..)

__What is the expected behaviour?__
(an option for) Renaming movies that will keep them part of the same movieset folder. 
I understand genre per movie is scraped from tmbd...   just wondering if there's anything that can be done ? maybe getting the most used genre from the set and placing it there (as an option) or so? 

I was at a friends house for xnas who'd manually sorted all his collection in genre, and that was cool to navigate (as we only wanted to watch scifi ;)...

**Comment by mkurios** [comment]
all the slashes got escaped, let me try again: 
`{$G\}{_Collections\$N\}$T { - $U }($Y)`

**Comment by mlaggner** [comment]
You have to make sure all movies have the same genre on the first place in the list - otherwise tmm has no chance to figure out what you want..
